### PR TITLE
TCOMP-1763 declare factory serializable

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TCOMP-1763
### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: component-runtime-impl/src/test/java/org/talend/sdk/component/runtime/record/RecordBuilderFactoryImplTest.java [hunk 1]
```diff
@@ -22,11 +22,16 @@
...
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.talend.sdk.component.api.record.Record;
 import org.talend.sdk.component.api.record.Schema;
 import org.talend.sdk.component.api.service.record.RecordBuilderFactory;
+import org.talend.sdk.component.runtime.serialization.ContainerFinder;
+import org.talend.sdk.component.runtime.serialization.DynamicContainerFinder;
+
+import java.io.*;
```
Comment (undx) [inline component-runtime-impl/src/test/java/org/talend/sdk/component/runtime/record/RecordBuilderFactoryImplTest.java:34]: avoid star import

**Review by undx — APPROVED** [review]
LGTM

File: component-runtime-impl/src/test/java/org/talend/sdk/component/runtime/record/RecordBuilderFactoryImplTest.java (hunk 1 above)
Comment (clesaec) [reply component-runtime-impl/src/test/java/org/talend/sdk/component/runtime/record/RecordBuilderFactoryImplTest.java]: done

**Review by undx — APPROVED** [review]
LGTM

**Pull request merged**